### ISSUE TITLE
GPX data seems consistently offset to the north

### ISSUE THREAD

**Issue by simplicitywebdesign** [mpetazzoni/leaflet-gpx#152]
![Screenshot 2024-03-10 at 23 09 53](https://github.com/mpetazzoni/leaflet-gpx/assets/10195471/21a3a492-cdae-494c-bce9-c0f305505d89)
The track shows successfully however the points aren't mapped accurately. There is a consistent offset. I have tried multiple GPX files from different sensors (Garmin _and_ iPhone) and the offset seems consistent at each zoom level. As you zoom in, the inaccuracy is lessened.
I'm at a loss to explain it. I'm still in local development but can provide code if needed, didn't post it to begin with as it's a GPX file so would be too big to paste in here. Suffice to say I'm not doing any processing of the GPX except for your script.

**Comment by mpetazzoni** [comment]
@simplicitywebdesign You can attach files to an issue by just drag-n-drop'ing them here. Would you mind share that GPX file so I can try for myself? This is very interesting indeed.

**Comment by mpetazzoni** [comment]
This could be a projection issue, which is more visible the further away from the equator you are. With that said, Tasmania is 41°S, and Northern California (where I'm at and I tested with some of my GPX from this area) is about 41°N, so no difference there 🤷🏻

Either that, or it's the basemap tiles that are off? Have you tried other layers?

**Comment by simplicitywebdesign** [comment]
Sure: https://assets.ryanmoore.au/gpx/tassie-combined.gpx

I am hoping it's something stupid I'm doing. It seems to be consistently 24–25 pixels to "the north"/top.

**Comment by mpetazzoni** [comment]
Confirmed it's an issue with the base layer, with the default OSM layer it's fine and aligned correctly:

<img width="803" alt="image" src="https://github.com/mpetazzoni/leaflet-gpx/assets/879507/aa398c88-9c10-422c-ae04-247b23efc088">

Also fine on the CyclOSM base map:

<img width="801" alt="image" src="https://github.com/mpetazzoni/leaflet-gpx/assets/879507/4c56001a-837b-40aa-87bb-d1f171ac947e">

**Comment by mpetazzoni** [comment]
@simplicitywebdesign Which base layer are you using in your example?

**Comment by simplicitywebdesign** [comment]
Hi Maxime, thanks for looking into this with me. You're right, it's definitely something off about the base layer, and in that spirit I had tried swapping out Thunderforest/OSM/etc without joy and gone down the rabbit hole of thinking my GPX coords were off, and I think I've skipped over the simpler possibility: a conflicting style or script on the site itself. I have set up a little sandbox with _just_ the map code and it displays fine. I'll have to go back and dig for the culprit.
